Geometry/HcalCommonData/python/testPhase1GeometryXML_cfi.py

```diff
@@ -6,7 +6,7 @@
     geomXMLFiles = cms.vstring('Geometry/CMSCommonData/data/materials.xml',
         'Geometry/CMSCommonData/data/rotations.xml',
         'Geometry/CMSCommonData/data/extend/cmsextent.xml',
-        'Geometry/CMSCommonData/data/cms/2019/v1/cms.xml',
+        'Geometry/CMSCommonData/data/cms/2021/v1/cms.xml',
```

Comment:
@boudoul - I'd argue that the xml files should not be moved. The year here indicates changes made this year, not a future scenario.

Reply:
@ianna could you please clarify what is your philosophy? Strictly speaking there is no 2019 detector, we have a 2018 and we will have a 2021. What is your concern?